applications/app/views/fragments/interactiveBody.scala.html

```diff
@@ -67,19 +66,4 @@
             @fragments.contentFooter(interactive, page.related, isPaidContent = isPaidContent)
         </div>
     }
-
-    @if(!Webpack.isSwitchedOn) {
-        <script>
```

Comment:
no i agree - inlining it and detaching it from the app should be more than enough?